index.html

```diff
@@ -154,7 +154,7 @@
                   <ul class="py-2 text-base text-gray-700 dark:text-gray-200" aria-labelledby="dropdownDefaultButton">
                     <li>
                       <button id="darkModeToggle" type="button" class="dropdownItem">
-                        Dark Mode: Off
+                        Dark Mode: Browser Preference
```

Comment:
Is it possible to rename this to just "System"? Seems more concise.

Reply:
Also a couple lines above in viewDropdown, there is a class `w-40`, which specifies the width of this dropdown. This should be just wide enough to make sure that Dark Mode fits without line wrapping.

Reply:
My intention was to be precise, because we actually follow the browser not the operating system. But system might be generic enough.

Reply:
Yeah. It's funny because I tested on Mac, safari and firefox browsers work fine. Browser preferences follow system. Chrome on the other hand...browser preferences do not align with system...

Reply:
So the whole Mode setting doesn't work in Chrome for you? Or does explicitly setting dark and light work on Chrome on macOS?

Reply:
Changed it to System and adjusted the width.